src/main/groovy/io/openliberty/tools/gradle/tasks/DeployTask.groovy

```diff
@@ -558,13 +558,15 @@ class DeployTask extends AbstractServerTask {
         try {
             Set<File> filesAsDeps = new HashSet<File>()
             //Get the compile and implementation dependencies that are included in the war
-            if (project.configurations.findByName('compile') != null) {
-                Set<File> compileDepFiles = project.configurations.compile.minus(project.configurations.providedCompile).getFiles()
-                filesAsDeps.addAll(compileDepFiles)
+            //compileClasspath: Use this to get all the dependencies required to compile your source code. It includes api, implementation, and compileOnly dependencies.
```

Comment:
There is one other place that we call `findByName('compile')` in AbstractServerTask. Should that change to use `compileClasspath`?